cpp/Ice/bidir/Client.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "MockAlarmClock.h"
 
+#include <Ice/Ice.h>
```

Comment:
We include Ice/Ice.h in Client and Server as the indirect inclusion from Chatbot.h and similar is not obvious.